Update EIP-725 (#5072)
ethereum/EIPs: EIPS/eip-725.md

---
eip: 725

> title: General data key/value store and execution standard

Reviewer: We're discouraging "standard" in titles and replaced "Simple Summary" with descriptions now:

```suggestion
title: General data key/value store and execution
description: An interface for a smart contract based account with attachable data key/value store
```

author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
discussions-to: https://github.com/ethereum/EIPs/issues/725
status: Review
type: Standards Track

> category: ERC
> requires: 165, 173

Reviewer: ```suggestion
category: ERC
created: 2017-10-02
requires: 165, 173
```

Header fields are out of order (and I can't grab the existing `created:` header in this suggestion :sweat_smile:)

> ## Simple Summary
> 

Reviewer: ```suggestion
```

> A standard interface for a smart contract based account with attachable data key/value store.

Reviewer: ```suggestion
```
